interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -20,87 +20,55 @@ if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
-    # will always depend on clangBasic).
-    DEPENDS clingInterpreter
-  )
-else()
-  set(_clad_build_type ${CMAKE_CFG_INTDIR})
-  if(APPLE)
-    set(_clad_extra_cmake_args
-      -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT}
-    )
-  endif()
+if(APPLE)
+  set(_clad_extra_cmake_args -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT})
+endif()
 
-  ExternalProject_Add(
-    clad
-    GIT_REPOSITORY https://github.com/vgvassilev/clad.git
-    GIT_TAG v0.7
-    UPDATE_COMMAND ""
-    CMAKE_ARGS -G ${CMAKE_GENERATOR}
-               -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-               -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-               -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
-               -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-               -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
-               -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
-               -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
-               -DLLVM_DIR=${LLVM_DIR}
-               -DClang_DIR=${Clang_DIR}
-               -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
-               -DCLAD_BUILD_STATIC_ONLY=ON
-               ${_clad_extra_cmake_args}
-    INSTALL_COMMAND env DESTDIR=${CMAKE_BINARY_DIR}/plugins ${CMAKE_COMMAND} --build . --target install
-               COMMAND  ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/plugins/${CMAKE_INSTALL_PREFIX}/ ${clad_install_dir}/plugins
-    BUILD_BYPRODUCTS ${CLAD_BYPRODUCTS}
-    # Wrap download, configure and build steps in a script to log output
-    LOG_DOWNLOAD ON
-    LOG_CONFIGURE ON
-    LOG_BUILD ON
-    LOG_INSTALL ON
-    # We need the target clangBasic to be built before building clad. However, we
-    # support building prebuilt clang and adding clangBasic breaks this case.
-    # Delegate the dependency resolution to the clingInterpreter target (which
-    # will always depend on clangBasic).
-    DEPENDS clingInterpreter
+ExternalProject_Add(
+  clad
+  GIT_REPOSITORY https://github.com/vgvassilev/clad.git
+  GIT_TAG v0.6
+  UPDATE_COMMAND ""
+  CMAKE_ARGS -G ${CMAKE_GENERATOR}
+             -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+             -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+             -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+             -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+             -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+             -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
+             -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+             -DLLVM_DIR=${LLVM_DIR}
+             -DClang_DIR=${Clang_DIR}
+             -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
+             -DCLAD_BUILD_STATIC_ONLY=ON
+             ${_clad_extra_cmake_args}
+  BUILD_COMMAND ${CMAKE_COMMAND} --build . --config ${_clad_build_type}
+  INSTALL_COMMAND ${_clad_dest_dir} ${CMAKE_COMMAND} --build . --config ${_clad_build_type} --target install
+                  COMMAND  ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/plugins/${CMAKE_INSTALL_PREFIX}/ ${clad_install_dir}/plugins
```

Comment:
That cannot work on Windows. ` ${CMAKE_BINARY_DIR}/plugins/${CMAKE_INSTALL_PREFIX}/` gives an invalid PATH:
```
Error copying directory from "C:/build/workspace/root-pullrequests-build/build/plugins/C:/build/workspace/root-pullrequests-build/install/" to "C:/build/workspace/root-pullrequests-build/build/etc/cling//plugins".
CMake Error at C:/build/workspace/root-pullrequests-build/build/interpreter/llvm/src/tools/cling/tools/plugins/clad/clad-prefix/src/clad-stamp/clad-install-Release-impl.cmake:19 (message):
  Command failed (1):

   'C:/Program Files/CMake/bin/cmake.exe' '-E' 'copy_directory' 'C:/build/workspace/root-pullrequests-build/build/plugins/C:/build/workspace/root-pullrequests-build/install/' 'C:/build/workspace/root-pullrequests-build/build/etc/cling//plugins'
```
As I already said, using simply `-DCMAKE_INSTALL_PREFIX=${clad_install_dir}/plugins` (or whatever) and `INSTALL_COMMAND ${CMAKE_COMMAND} --build . --config ${_clad_build_type} --target install` works fine.

Reply:
Thanks! I think now it is resolved.

Reply:
Did you set `-DCMAKE_INSTALL_PREFIX=${clad_install_dir}` (or `-DCMAKE_INSTALL_PREFIX=${clad_install_dir}/plugins`)? Otherwise it will not be in the right location...

Reply:
Doh, you are right but that may allow us to get rid of the custom install command for unix..